videogame-tiled.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * videogame-tiled.js
+ *
+ * This extension adds a 'videogame map format' (Ctrl+F) type to the Export As
+ * menu, which can be used to generate matrix maps as used by the Scene.build()
+ * method from the videogame library (https://github.com/diogoeichert/videogame)
+ */
+
+tiled.registerMapFormat("videogame", {
+    name: "videogame map format",
+    extension: "json",
+
+    write: (map, fileName) => {
+        for (let i = 0; i < map.layerCount; ++i) {
+            const layer = map.layerAt(i);
+
+			if (!layer.isTileLayer) {
+				continue;
+            }
+
+			var file = new TextFile(fileName, TextFile.WriteOnly);
+			file.write("[\n");
+	
+			for (let y = 0; y < layer.height; ++y) {
+				const row = [];
+
+				for (let x = 0; x < layer.width; ++x) {
+					row.push(`"${FileInfo.baseName(layer.tileAt(x, y).imageFileName)}"`);
```

Comment:
`layer.tileAt(x, y)` may return `null` when there is no tile at the given location, a case which should probably be handled.